.env.example

```
@@ -1,27 +1,30 @@
+#   cp .env.example .env
+
+# PostgreSQL (defaults work for local development)
 POSTGRES_USER=sandbox
 POSTGRES_PASSWORD=sandbox_dev_password
 POSTGRES_DB=postgis
 POSTGRES_PORT=5439
 
+# Cloudflare R2 (S3-compatible object store)
+# Create a bucket and API token at https://dash.cloudflare.com/ → R2
+R2_ACCOUNT_ID=your_account_id
+R2_ACCESS_KEY_ID=your_access_key_id
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The R2_ACCESS_KEY_ID key should go before the R2_ACCOUNT_ID key

(UnorderedKey)